.github/workflows/conformance-kind-proxy-embedded.yaml

```diff
@@ -124,6 +124,30 @@ jobs:
           title: "Summary of all features tested"
           json-filename: "${{ env.job_name }}"
 
+      - name: Install Cilium with secret sync disabled
+        id: install-cilium-secret-sync-disabled
+        run: |
+          helm upgrade cilium ./install/kubernetes/cilium \
+            --namespace kube-system \
+            --reuse-values \
+            --set=tls.secretSync.enabled=false
+
+          kubectl -n kube-system rollout restart ds/cilium deployment/cilium-operator
+          cilium status --wait --wait-duration=10m
+          kubectl -n kube-system get pods
+
+      - name: Run L7 related connectivity test
+        run: |
+          cilium connectivity test --test="l7|sni|check-log-errors" \
+            --junit-file "cilium-junits/${{ env.job_name }}-without-secret-sync.xml" \
+            --junit-property github_job_step="Run connectivity test with secret sync disabled"
```

Comment:
@brlbil @cilium/ci-structure I would expect CI structure review to catch that this workflow was not reusing `${{ steps.vars.outputs.connectivity_test_defaults }}` and hence introduced a few CI bugs here such as failing to capture sysdump on failure.

Fix is here: https://github.com/cilium/cilium/pull/41036